aws/macie.go

```diff
@@ -0,0 +1,87 @@
+package aws
+
+import (
+	goerror "errors"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/macie2"
+	"github.com/aws/aws-sdk-go/service/sts"
+	"github.com/gruntwork-io/cloud-nuke/config"
+	"github.com/gruntwork-io/cloud-nuke/logging"
+	"github.com/gruntwork-io/go-commons/errors"
+)
+
+func getAllMacieMemberAccounts(session *session.Session, excludeAfter time.Time, configObj config.Config) ([]string, error) {
```

Comment:
Yeah - unfortunately we can't really implement this pattern with Macie member accounts. Since account membership is determined by the invitation acceptance status, and since the API doesn't expose when an account last accepted an invite, we don't have much to go off.

Reply:
Perhaps remove it from the arg list and add a comment indicating that we can't support it?